certified-connectors/The Bot Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,329 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "The Bot Platform",
+		"description": "Connect The Bot Platform's bots to any application, send messages to bot users, set attribute values using your existing bots at The Bot Platform",
+		"version": "1.0",
+		"contact": {
+			"name": "The Bot Platform Support",
+			"url": "https://thebotplatform.com/https-thebotplatform-com-contact/",
+			"email": "hello@thebotplatform.com"
+		}
+	},
+	"host": "api.thebotplatform.com",
+	"basePath": "/",
+	"schemes": [
+		"https"
+	],
+	"x-ms-connector-metadata": [{
+			"propertyName": "Website",
+			"propertyValue": "https://thebotplatform.com"
+		},
+		{
+			"propertyName": "Privacy policy",
+			"propertyValue": "https://thebotplatform.com/privacy-policy/"
+		},
+		{
+			"propertyName": "Categories",
+			"propertyValue": "Communication;Website"
+		}
+	],
+	"consumes": [],
+	"produces": [],
+	"paths": {
+		"/v1.0/activity/external": {
+			"post": {
+				"responses": {
+					"201": {
+						"description": "created",
+						"schema": {
+							"type": "object",
+							"properties": {
+								"data": {
+									"$ref": "#/definitions/ExternalActivity"
```

Comment:
@tbpsami The schema reference '#/definitions/ExternalActivity' could not be found in the swagger document. Please define the reference in the definition section.

Reply:
For all the schema references, is it possible to link them to a hyperlink instead of having the definition inside the swagger file? @Amjed-Ayoub

Reply:
can you please clarify more?